app/javascript/components/patient/assessment/actions/MonitoringPeriod.js

```diff
@@ -0,0 +1,90 @@
+import React from 'react';
+import { PropTypes } from 'prop-types';
+import { Col, Row } from 'react-bootstrap';
+import moment from 'moment';
+import { formatDate } from '../../../../utils/DateTime';
+
+import ExtendedIsolation from './ExtendedIsolation';
+import InfoTooltip from '../../../util/InfoTooltip';
+import LastDateExposure from './LastDateExposure';
+import SymptomOnset from './SymptomOnset';
+
+class MonitoringPeriod extends React.Component {
+  constructor(props) {
+    super(props);
+  }
+
+  formatEndOfMonitoringDate = () => {
+    const eom = this.props.patient.linelist.end_of_monitoring;
+    return eom === 'Continuous Exposure' ? eom : formatDate(eom);
+  };
+
+  render() {
+    return (
+      <Row>
+        <Col sm={8}>
```

Comment:
These columns get wonky on small screens.  I'd have these be dynamic based on workflow, like at smaller breakpoints have SO and LDE each be 50% width.  Would have to do something for exp too

Reply:
This will also need to be fixed:
![Screen Shot 2021-06-15 at 1 29 04 PM](https://user-images.githubusercontent.com/35042815/122097546-cc100300-cddd-11eb-88f3-44a5a8a95e1f.png)

I just did something similar, lmk if you need help

Reply:
fixed it by fixing the pagination layout: https://github.com/SaraAlert/SaraAlert/pull/861/commits/0b00f9efd83684aa21526ddee93b433c20c8fb79#diff-229373df9dabbd63465b0a51e5b7fdf3ac5bab800c7898975b878e988f2bc3c0R264